src/euphorie/client/tests/test_model.py

```diff
@@ -387,14 +393,15 @@ def testSessions(self):
         group2.parent = group1
         from functools import partial
 
-        add_survey = partial(model.SurveySession, account=account1)
-        survey1 = add_survey(zodb_path="1")
-        session.add(survey1)
-        survey2 = add_survey(zodb_path="2", group=group1)
-        session.add(survey2)
-        survey3 = add_survey(zodb_path="3", group=group2)
-        session.add(survey3)
-        session.flush()
+        with mock.patch("plone.api.user.get_current", return_value=nobody):
```

Comment:
I think that in this case the return_value should be sqlalchemy account, i.e. `account1`.

Reply:
I have updated the PR to do this.  But then the tests actually fail again: in the `osha.oira` subscriber `get_current_account` then gets an actual account, and then the subscriber [tries to get the portal](https://github.com/euphorie/osha.oira/blob/e36ebf46053519eabad4d425cf8fe6370c1756c9/src/osha/oira/client/subscribers.py#L42) and this fails.

So maybe we should abandon this, and instead catch the error in `osha.oira` as you suggest.  I will push a PR for that.